packages/block-editor/src/components/rich-text/index.js

```diff
@@ -269,7 +269,7 @@ export function RichTextWrapper(
 		readOnly || disableBoundBlock || shouldDisableForPattern;
 
 	const { getSelectionStart, getSelectionEnd, getBlockRootClientId } =
-		useSelect( blockEditorStore );
+		useSelect( blockEditorStore, [] );
```

Comment:
This is not correct, useSelect without any other arguments than the store name, doesn't need a second or third argument.

Reply:
Yup - I found that out too, but forgot to finish the review, apparently.
<img width="832" height="598" alt="" src="https://github.com/user-attachments/assets/4c7d5abb-2c39-4534-9436-c1dc706c381f" />

Let me know if you have any leads for performance improvements. I haven't found anything reliable yet.

Reply:
Yes, the static getters doesn't need a second argument.